### ISSUE TITLE
Initialisation Error - Windows 10, new install of code and extension

### ISSUE THREAD

**Issue by SwarfegaGit** [PowerShell/vscode-powershell#1847]
The extension fails to load the console.  It takes a while before it times out with an initialisation error.  The logs just show...

```
05/04/2019 07:46:50 [NORMAL] - Visual Studio Code v1.33.0 64-bit
05/04/2019 07:46:50 [NORMAL] - PowerShell Extension v1.12.0
05/04/2019 07:46:50 [NORMAL] - Operating System: Windows 64-bit
05/04/2019 07:46:50 [NORMAL] - Language server starting --
05/04/2019 07:46:50 [NORMAL] -     exe: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
05/04/2019 07:46:50 [NORMAL] -     args: C:\Users\ME\.vscode\extensions\ms-vscode.powershell-1.12.0\modules\PowerShellEditorServices\Start-EditorServices.ps1 -HostName 'Visual Studio Code Host' -HostProfileId 'Microsoft.VSCode' -HostVersion '1.12.0' -AdditionalModules @('PowerShellEditorServices.VSCode') -BundledModulesPath 'C:\Users\ME\.vscode\extensions\ms-vscode.powershell-1.12.0\modules' -EnableConsoleRepl -LogLevel 'Normal' -LogPath 'C:\Users\ME\.vscode\extensions\ms-vscode.powershell-1.12.0\logs\1554446810-5aaeda74-9b5e-4ab8-bf2c-7c6a614b098a1554446804466\EditorServices.log' -SessionDetailsPath 'C:\Users\ME\.vscode\extensions\ms-vscode.powershell-1.12.0\sessions\PSES-VSCode-1292-974954' -FeatureFlags @()
05/04/2019 07:46:53 [NORMAL] - powershell.exe started, pid: 11356
05/04/2019 07:46:54 [NORMAL] - powershell.exe terminated or terminal UI was closed
05/04/2019 07:48:51 [NORMAL] - Language server startup failed.
05/04/2019 07:48:51 [ERROR] - The language service could not be started: 
05/04/2019 07:48:51 [ERROR] - Timed out waiting for session file to appear.
```

I can see people have had this issue in the past but they appear to be fixed on releases.   This is a fresh install of Windows 10 along with Code and the extension.  This is a company laptop though on Azure AD so something tells me they might have changed something.

PowerShell 6 will open fine.
Execution policy is RemoteSigned

Tried the preview extension but also fails.

**Comment by TylerLeonhardt** [comment]
Thanks for the info! Can you also [attach the logs](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#providing-information-about-your-environment) here so we can get a better idea of what's going on?

**Comment by SwarfegaGit** [comment]
I've done this but the logs seem pretty empty.

````
10/04/2019 08:57:41 [NORMAL] - Language server starting --
10/04/2019 08:57:41 [NORMAL] -     args: C:\Users\MyUser\.vscode\extensions\ms-vscode.powershell-1.12.0\modules\PowerShellEditorServices\Start-EditorServices.ps1 -HostName 'Visual Studio Code Host' -HostProfileId 'Microsoft.VSCode' -HostVersion '1.12.0' -AdditionalModules @('PowerShellEditorServices.VSCode') -BundledModulesPath 'C:\Users\MyUser\.vscode\extensions\ms-vscode.powershell-1.12.0\modules' -EnableConsoleRepl -LogLevel 'Diagnostic' -LogPath 'C:\Users\MyUser\.vscode\extensions\ms-vscode.powershell-1.12.0\logs\1554883061-bc0f58bb-fb8e-4361-a206-db7459a967a51554883053735\EditorServices.log' -SessionDetailsPath 'C:\Users\MyUser\.vscode\extensions\ms-vscode.powershell-1.12.0\sessions\PSES-VSCode-16512-367824' -FeatureFlags @()
10/04/2019 08:57:41 [NORMAL] -     exe: C:\Windows\SysWow64\WindowsPowerShell\v1.0\powershell.exe
10/04/2019 08:57:43 [NORMAL] - powershell.exe started, pid: 9584
10/04/2019 08:57:44 [NORMAL] - powershell.exe terminated or terminal UI was closed
10/04/2019 08:59:42 [NORMAL] - Language server startup failed.
10/04/2019 08:59:42 [ERROR] - The language service could not be started: 
10/04/2019 08:59:42 [ERROR] - Timed out waiting for session file to appear.
````

**Comment by TylerLeonhardt** [comment]
Can you set the log level to "Diagnostic" and attach _those_ logs? It should create a `Start-EditorServices` log file.

setting:
```
"powershell.developer.editorServicesLogLevel": "Diagnostic"
```

The link above has the full steps.

**Comment by SwarfegaGit** [comment]
There are no logs though.  Whilst the argument suggests there should be a EditorServices.log file there isn't one.

````
11/04/2019 14:34:39 [NORMAL] - Visual Studio Code v1.33.0 64-bit
11/04/2019 14:34:39 [NORMAL] - PowerShell Extension v1.12.0
11/04/2019 14:34:39 [NORMAL] - Operating System: Windows 64-bit
11/04/2019 14:34:39 [NORMAL] - Language server starting --
11/04/2019 14:34:39 [NORMAL] -     exe: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
11/04/2019 14:34:39 [NORMAL] -     args: C:\Users\MyAccount\.vscode\extensions\ms-vscode.powershell-1.12.0\modules\PowerShellEditorServices\Start-EditorServices.ps1 -HostName 'Visual Studio Code Host' -HostProfileId 'Microsoft.VSCode' -HostVersion '1.12.0' -AdditionalModules @('PowerShellEditorServices.VSCode') -BundledModulesPath 'C:\Users\MyAccount\.vscode\extensions\ms-vscode.powershell-1.12.0\modules' -EnableConsoleRepl -LogLevel 'Diagnostic' -LogPath 'C:\Users\MyAccount\.vscode\extensions\ms-vscode.powershell-1.12.0\logs\1554989679-9cab73cf-187c-4687-a58d-ae95c0d9adc01554989674668\EditorServices.log' -SessionDetailsPath 'C:\Users\MyAccount\.vscode\extensions\ms-vscode.powershell-1.12.0\sessions\PSES-VSCode-2336-805766' -FeatureFlags @()
11/04/2019 14:34:40 [NORMAL] - powershell.exe started, pid: 14196
11/04/2019 14:34:41 [NORMAL] - powershell.exe terminated or terminal UI was closed
11/04/2019 14:36:39 [NORMAL] - Language server startup failed.
11/04/2019 14:36:39 [ERROR] - The language service could not be started: 
11/04/2019 14:36:39 [ERROR] - Timed out waiting for session file to appear.
````

````
PS C:\Users\MyAccount> Get-ChildItem 'C:\Users\A548059\.vscode\extensions\ms-vscode.powershell-1.12.0\logs\1554989679-9cab73cf-187c-4687-a58d-ae95c0d9adc01554989674668'


    Directory: C:\Users\MyAccount\.vscode\extensions\ms-vscode.powershell-1.12.0\logs\1554989679-9cab73cf-187c-4687-a58d-ae95c0d9adc01554989674668


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----       11/04/2019     14:36           1271 vscode-powershell.log


PS C:\Users\MyAccount>
````

**Comment by TylerLeonhardt** [comment]
@SwarfegaGit do you use an anti virus? We've received a lot of reports in the last version that antivirus has been getting in the way of PowerShell Editor Services starting up in the background.

We have a fix that is pending a new release.

**Comment by SwarfegaGit** [comment]
Yes, McAfee Endpoint Security.  
I think you're correct as looking in the Application Event Log after opening VSCode I see this...
````
Log Name:      Application
Source:        McAfee Endpoint Security
Date:          12/04/2019 10:24:08
Event ID:      3
Task Category: None
Level:         Error
Keywords:      Classic
User:          SYSTEM
Computer:      --------------
Description:
EventID=18054

ExP:Illegal API Use Blocked an attempt to exploit C:\WINDOWS\SYSTEM32\WINDOWSPOWERSHELL\V1.0\POWERSHELL.EXE, which targeted the AtlComPtrAssign API.
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="McAfee Endpoint Security" />
    <EventID Qualifiers="8192">3</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2019-04-12T09:24:08.106539300Z" />
    <EventRecordID>5344</EventRecordID>
    <Channel>Application</Channel>
    <Computer>DESKTOP-L799GUR</Computer>
    <Security UserID="S-1-5-18" />
  </System>
  <EventData>
    <Data>EventID=18054

ExP:Illegal API Use Blocked an attempt to exploit C:\WINDOWS\SYSTEM32\WINDOWSPOWERSHELL\V1.0\POWERSHELL.EXE, which targeted the AtlComPtrAssign API.</Data>
  </EventData>
</Event>
````

**Comment by TylerLeonhardt** [comment]
This is fixed in #1837. Pending a new release which should happen today or early next week!

**Comment by TylerLeonhardt** [comment]
I will dupe this to https://github.com/PowerShell/vscode-powershell/issues/1831. We've already got a report that says the fix worked.
